pkg/file/redundancy/level.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package redundancy
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+// Level is the redundancy level
+// which carries information about how much redundancy should be added to data to remain retrievable with a 1-10^(-6) certainty
+// in different groups of expected chunk retrival error rates (level values)
+type Level uint8
+
+const (
+	// no redundancy will be added
+	NONE Level = iota
+	// expected 1% chunk retrieval error rate
+	MEDIUM
+	// expected 5% chunk retrieval error rate
+	STRONG
+	// expected 10% chunk retrieval error rate
+	INSANE
+	// expected 50% chunk retrieval error rate
```

Comment:
This number seems to be incorrect, we had a much higher number when performing the tests. The same goes for the above.

Reply:
what do you mean by "much higher number"? you got errors only with higher retrieval error rate?

Reply:
@mrekucci i think this 50% is the error rate for not being able to find chunks, not the data growth percentage